check_copyright_config.yaml

```diff
@@ -0,0 +1,32 @@
+DEFAULT:
+  perform_check: yes  # Ensure that license header check is performed
+
+  # List of allowed licenses for files in this section
+  allowed_licenses:
+    - Apache-2.0
+    - MIT
+  license_for_new_files: Apache-2.0  # License to be used when inserting a new copyright notice
+
+  new_notice_python: |
+    """
+    SPDX-FileCopyrightText: © {years} Tenstorrent AI ULC
+
+    SPDX-License-Identifier: {license}
+    """
+
+  # Define the template for replacing comment lines with SPDX notation
+  espressif_copyright: '© {years} Tenstorrent AI ULC'
+
+  # Update the include section with the file types you want to check
+  include:
+    - "*.py"
+
+ignore:  # You can also select ignoring files here
+  perform_check: no  # Don't check files from that block
+  include:
+    - .github/*
+    - __pycache__/*
+    - /CONTRIBUTING.md
+    - /MAINTAINERS.md
+    - /model_demos/cv_demos/yolo_v3/holli_src/*
+    - /model_demos/nlp_demos/falcon/utils/*
```

Comment:
From this directory, `model_demos/nlp_demos/falcon/utils/configuration_RW.py` should remain unchanged as it already has the source header.

The other files will contain the Tenstorrent header as they are our own generated files.